src/compiler/transformers/declarations.ts

```diff
@@ -984,7 +986,8 @@ export function transformDeclarations(context: TransformationContext) {
                 decl.modifiers,
                 decl.importClause,
                 rewriteModuleSpecifier(decl, decl.moduleSpecifier),
-                getResolutionModeOverrideForClauseInNightly(decl.assertClause),
+                getResolutionModeOverrideInNightly(decl.assertClause),
+                getResolutionModeOverrideInNightly(decl.attributes),

```

Comment:
We can remove the nightly-only restriction as part of this change 🎉

Reply:
Oke, will make a cleanup...